### PR TITLE
Trip and lockout hvdc lines

### PR DESCRIPTION
**Pull request by ghazwarhili**

### CODE REVIEW

**Review by sBouzols — CHANGES_REQUESTED** [review]
Could you add some UT please for those new cases ?
Will test

**Review by EstherDarkish — CHANGES_REQUESTED** [review]
java.lang.IllegalArgumentException: No enum constant org.gridsuite.modification.server.ModificationType.BRANCH_STATUS_MODIFICATION

When trying to display the list of modifications of my node in the frontend

**Review by EstherDarkish — APPROVED** [review]
Ok

File: src/test/java/org/gridsuite/modification/server/modifications/OperatingStatusModificationLockoutHvdcLineTest.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    @Override
+    protected void assertAfterNetworkModificationCreation() {
+        TestUtils.assertOperatingStatus(getNetwork(), TARGET_HVDC_LINE_ID, TARGET_HVDC_LINE_STATUS);
+    }
+
+    @Override
+    protected void assertAfterNetworkModificationDeletion() {
+        // go back to init status
+        TestUtils.assertOperatingStatus(getNetwork(), TARGET_HVDC_LINE_ID, TARGET_HVDC_LINE_STATUS);
+    }
```
[the remark this answers is not in the document]
Comment (sBouzols) [reply src/test/java/org/gridsuite/modification/server/modifications/OperatingStatusModificationLockoutHvdcLineTest.java:67]: Maybe we should add an assertNoOperatingStats which check if the extension is absent ?

[the remark this answers is not in the document]
Comment (sBouzols) [reply src/test/java/org/gridsuite/modification/server/modifications/OperatingStatusModificationLockoutHvdcLineTest.java:67]: add an assert to check if all terminals are connected ?

**Review by sBouzols — APPROVED** [review]
Code Review OK

**Pull request merged**